Display the number transaction in Posting Journal

lomamech opened the pull request:
Display the number transaction in Footer of Posting Journal

closes #1551

lomamech committed:
Fix test E2E for edit journal transaction

lomamech committed:
Adding the number of transanction from Journal

lomamech committed:
Adding test integration for path /journal/count

lomamech committed:
Resolve conflict with branch Master

lomamech committed:
Sanitaze code

sfount commented on client/src/modules/journal/journal.js:
```diff
@@ -312,14 +322,20 @@ function JournalController(Journal, Sorting, Grouping,
-
+        vm.gridOptions.showGridFooter = true;
+        vm.gridOptions.gridFooterTemplate = '<div><strong>' + $translate.instant('FORM.INFO.NUM_TRANSACTION') + 
+          ' : ' + transactionNumber + ' / ' + vm.numberTransaction + '</strong></div>'; 
```
@lomamech can you rename these variables to something that provides more context? There is no real difference between `numberTransaction` and `transactionNumber` - this might be hard to follow when revisiting this code in the future. 

For example: 
```js
// number of all of the transactions in the system
var numberTotalSystemTransactions

// number of transactions downloaded and shown in the current journal 
var numberCurrentGridTransactions
```

sfount reviewed (commented):
This change looks good to me! if possible could you include a quick screenshot of the feature that is added here to get an idea of how it works along with this PR?

lomamech committed:
Rename variable for
-get of all of the transactions in the System
-get of transactions downloaded and shown in the current journal
